<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="25474" started-at="2020-02-13T07:07:12Z" finished-at="2020-02-13T07:07:37Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="25474" started-at="2020-02-13T07:07:12Z" finished-at="2020-02-13T07:07:37Z">
      <class name="qbrainx.executable.scripts.LoginScripts">
        <test-method status="PASS" signature="beforeExecution()[pri:0, instance:qbrainx.executable.scripts.LoginScripts@2ff4f00f]" name="beforeExecution" is-config="true" duration-ms="13625" started-at="2020-02-13T12:37:12Z" finished-at="2020-02-13T12:37:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeExecution -->
        <test-method status="PASS" signature="testCase1(java.lang.String, java.lang.String)[pri:0, instance:qbrainx.executable.scripts.LoginScripts@2ff4f00f]" name="testCase1" duration-ms="4455" started-at="2020-02-13T12:37:26Z" data-provider="testdatas" finished-at="2020-02-13T12:37:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[bbcbdc]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[usbchsbcu]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase1 -->
        <test-method status="PASS" signature="testCase1(java.lang.String, java.lang.String)[pri:0, instance:qbrainx.executable.scripts.LoginScripts@2ff4f00f]" name="testCase1" duration-ms="7347" started-at="2020-02-13T12:37:30Z" data-provider="testdatas" finished-at="2020-02-13T12:37:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chvhdsv]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[bcudsbcub]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase1 -->
        <test-method status="FAIL" signature="testCase2()[pri:0, instance:qbrainx.executable.scripts.LoginScripts@2ff4f00f]" name="testCase2" duration-ms="34" started-at="2020-02-13T12:37:37Z" finished-at="2020-02-13T12:37:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ Home page of FB  expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError:  Home page of FB  expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at qbrainx.executable.scripts.LoginScripts.testCase2(LoginScripts.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase2 -->
      </class> <!-- qbrainx.executable.scripts.LoginScripts -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
